examples/websub-internal-hub-sample/subscriber.bal

```diff
@@ -20,11 +20,11 @@ endpoint websub:Listener websubEP {
 }
 service websubSubscriber bind websubEP {
 
-    // Resource accepting intent verification requests.
-    // If this resource is not specified, intent verification would happen automatically - intent would be verified if
-    // the topic specified in the intent verification request matches that specified in the annotation.
+    // Define the resource that accepts the intent verification requests.
+    // If the resource is not specified, intent verification happens automatically. 
+    // It verifies if the topic specified in the intent verification request matches the topic specified as the annotation. 
```

Comment:
Can we have the content in line 25 start at the end of line 24? Because line 25 is relevant only if the resource is not specified.

Reply:
ack! Sent a commit